wp-content/themes/wp-starter/README.md

```diff
@@ -24,6 +24,21 @@ Blocks are build using ACF and core WordPress blocks. Styles for the blocks are
 * Text Image
 * Video Embed
 
+### Creating New Blocks
+The theme is set up with [plop](https://plopjs.com/) which will auto generate a new block based on the options you input. 
+In order build a new block run:
+
+```
+ddev npm run plop block
```

Comment:
One thing that would be cool would be to set up a simple [custom command](https://ddev.readthedocs.io/en/stable/users/extend/custom-commands/) in `ddev` like `ddev create-block` that can be run anywhere in the project, that automatically executes from the theme directory.

Reply:
Ah, that would be nice. I will look into that.